CONTRIBUTING.md

```diff
@@ -0,0 +1,214 @@
+# Contribute to the secrets of sonar-text
+
+## Development environment
+
+1. Set up a Java development environment, or use Docker, to be ready to perform commands such as those specified in the [README.md](README.md)
+2. Set up an extension that validates YAML files, such as [YAML by RedHat](https://github.com/redhat-developer/vscode-yaml)
+3. Set up this [json-schema file](https://github.com/SonarSource/sonar-text/blob/master/sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/specifications/specification-json-schema.json) for your validation tooling
+4. Take a look at [the existing rules](https://github.com/SonarSource/sonar-text/tree/master/sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration)
+5. Familiarize yourself with [SonarSource/rspec](https://github.com/SonarSource/rspec)
+
+## Adding a secret
+
```

Comment:
Generate skeletons of RSPEC and YAML files:
```
python bootstrap_secret.py -v -R ./rspec -T ./sonar-text -n XXXX Provider "Category" 
```

WARNING: This will generate a new RSPEC ID.

Reply:
where is this script located please ?

Reply:
It is in our (LT) private repository. I am not sure I can post internal information on a public page.

Reply:
I will add this information to our internal wiki then instead of here